yeoman-generator/index.d.ts

```diff
@@ -68,6 +68,7 @@ declare class Base extends EventEmitter {
     appname: string;
     config: Base.Storage;
     fs: Base.MemFsEditor;
+    options: any;
```

Comment:
You can make the type more specific in your class:

```typescript
class MyGenerator extends Base {
  options: {
    foo: boolean;
  };
}
```

But using `{}` prevents the following:

```
class MyGenerator extends Base {
  options: string; // options is never a string.
}
```